datafusion/physical-plan/src/projection.rs

```diff
@@ -56,6 +56,8 @@ pub struct ProjectionExec {
     input: Arc<dyn ExecutionPlan>,
     /// The output ordering
     output_ordering: Option<Vec<PhysicalSortExpr>>,
```

Comment:
In theory this doesn't need to be stored now, since it can be obtained via `self.equivalence_properties.oeq_class().output_ordering()`, however the last method has some non-trivial computation which are probably best avoided anytime there's a call to `ProjectionExec::output_ordering()`